### PR TITLE
[WIP-Outreachy Task]: [Send email module]

### PR DESCRIPTION
**Pull request by AbrarEmad**
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/pybind/mgr/sendemail/module.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    def handle_command(self, inbuf, cmd):
+
+        self.log.error("hello_world_error")
+        smtp_ssl_host = 'smtp.gmail.com'  # smtp.mail.yahoo.com
+	smtp_ssl_port = 465
+	sender = 'add Your Email here'
+	password = 'add Your Password here'
+	target = 'add the target email here'
+
+	msg = MIMEText('Hi, how are you today?')
```
Comment (liewegas) [inline src/pybind/mgr/sendemail/module.py]: Try making the body a dump of self.get('health') (e.g., 'heatlh status:\n%s' % self.get('health'))

**Comment by liewegas** [comment]
You only need to include 2 files: the __init__.py and module.py.  The other files appear to be remannts from.. a pip install into the pybind directory maybe?  (I'm not sure, I'm not a python expert.)  You can use git gui and then amend the commit and unstage those files.  Or git rm them and git commit --amend..

**Comment by AbrarEmad** [comment]
Thanks i will do that , and about health command am i adding it in code or
run it in terminal?
is there any document for this?

Thanks in advance,
Abrar

**Comment by liewegas** [comment]
Thanks for your interest in Outreachy!

**Pull request closed without merging**